vllm/model_executor/models/utils.py

```diff
@@ -172,8 +173,9 @@ def _load_module(
         if module != self.module:
             module_load_weights = getattr(module, "load_weights", None)
             if callable(module_load_weights):
-                module_load_weights(weights)
-                return
+                loaded_params = module_load_weights(weights)
+                yield from map(lambda x: self._get_qualname(base_prefix, x),
+                               loaded_params)
```

Comment:
I also modified `load_weights` in most of models to return loaded params' names for alignment and compatibility, even though they aren't used with `AutoWeightsLoader`.